services/database/user.ts

```diff
@@ -35,4 +42,177 @@ async function getUsers() {
     return users;
 }
 
-export { getUser, getUsers };
+/* 
+TODO (6/30/21): test passing userId in newTeacherData instead of through connect key
+*/
+
+async function updateUser(user: UserInput): Promise<User> {
+    /*
+    Flow:
+    - get role and role data from UserInput
+    - create/update the role record
+    - update the user record
+    - return the user, including the role records
+    */
+    const roleData = user.role_data;
```

Comment:
After we add the role field in the user database, let's set that field in this function

Reply:
https://github.com/uwblueprint/social-diversity-for-children/pull/18/commits/49a165655b1cf4bf9b60d849a1cb1baf7c17aa6b